db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20171205103643) do
+ActiveRecord::Schema.define(version: 20171206172721) do
```

Comment:
Not so sure if we should merge this file into dev (because you haven't changed anything db relevant), maybe override it when updating your branch onto the latest dev?